googleMapsApiModule.py

```diff
@@ -34,3 +36,13 @@ def mapsstatic(search_location):
     result_url = f'https://maps.googleapis.com/maps/api/staticmap?center={address.lower().replace(" ", "+")}&zoom=13&scale=1&size=600x350&maptype=roadmap&key={config.key}&format=png&visual_refresh=true&markers=size:mid%7Ccolor:0xff0000%7Clabel:L%7C{address.lower().replace(" ", "+")}'
     logging.debug(result_url)
     webbrowser.open_new(result_url)
+
```

Comment:
Also, add the logging decorator here as well.

Reply:
I've made the changes, please review.